py/sentry_relay/consts.py

```diff
@@ -1,25 +1,93 @@
+from enum import IntEnum
+
+from sentry_relay._compat import int_enum_base
```

Comment:
Can we just have a regular class? I think it's also problematic that we change behavior drastically between py2 vs 3. What's the purpose of subclassing from int?

Reply:
Also I don't understand why we use `int_enum_base` for one enum and enum34 for the other.

Reply:
`DataCategory` was an `IntEnum` in Sentry already, and we rely on the behavior that's also reflected in tests, for example:

```py
1 == DataCategory.ERROR
```

We *also* need the class methods for backwards compatibility (and generally as part of the usage of `DataCategory`. Now, the problem is that there seems to be incompatibility between Python 2 and 3. If I make the base class `int` in Python 3, it cannot create the enum. If I make the base class `object` (or anything else) in Python 2, it cannot create the enum.

Reply:
Maybe this is not actually a difference between Python 2 and 3, but rather between builtin `enum` and `enum34`.

Reply:
If you only care about the equality stuff (and not the actual type relations) I'd recommend implementing the `__eq__` method.

Reply:
Actually ignore, `IntEnum` already subclasses from `int`.

Reply:
https://bitbucket.org/stoneleaf/enum34/src/c208549a93b71b948ff7bbdfd29dce8f85527916/enum/__init__.py#lines-819

Reply:
Thanks, updated.